clients/bot_client.go

```diff
@@ -54,6 +59,9 @@ func (botClient *BotClient) InitOlmMachine(client *mautrix.Client, nebStore *mat
 
 	botClient.stateStore = &NebStateStore{&nebStore.InMemoryStore}
 	olmMachine := crypto.NewOlmMachine(client, cryptoLogger, cryptoStore, botClient.stateStore)
+	olmMachine.AcceptVerificationFrom = func(_ string, _ *crypto.DeviceIdentity) (crypto.VerificationRequestResponse, crypto.VerificationHooks) {
+		return crypto.AcceptRequest, botClient
```

Comment:
I wonder if we should have a whitelist here rather than allowing anyone to do this...

Reply:
Could be a regex on the user ID in the config?

Reply:
regexes SGTM

Reply:
Sure, added the regexes functionality.